backend/docker-compose-bare-metal.yaml

```diff
@@ -0,0 +1,196 @@
+x-bare-metal-common-environment: &common-environment
```

Comment:
Yaml Lessons Learned: 
- `x-` naming is required for Custom Properties in Docker compose
- Object format (map) cannot be merged (`<<:`) with List format
- Strings with some special characters must be `'` or `"` delimited, simple strings (words) do not need to be delimited
- Comments inside maps seems to break them

Reply:
YAML: Yet Another Masochistic Language